src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -120,8 +123,33 @@ class UserProfileAdd extends Component {
   componentDidMount() {
     this.state.showphone = true;
     this.onCreateNewUser();
+    this.fetchTeamCodeAllUsers(); 
```

Comment:
This is an `async` function, and since we wanted to load it alongside the modal, we better `await` it.